### DISCUSSION TITLE
Legacy File Transfer Dialog Not Working

### DISCUSSION THREAD

**Discussion by christopbader** [valinet/ExplorerPatcher#3951] [General]
![Screenshot (60)](https://github.com/user-attachments/assets/bdd0504d-9d47-45e3-887e-b6a89ebd2af9)
Legacy transfer dialog has stopped working for me and its now on the Win-11 dialog
![Winver 14-11-24](https://github.com/user-attachments/assets/8c520ac2-5b0f-4c8f-a720-0f49df83908d)
![Windows -11](https://github.com/user-attachments/assets/28aaeab7-4ef3-48ab-8852-8ffc681256fd)
![Legacy](https://github.com/user-attachments/assets/40829952-e1de-4bb5-a6f2-a0de5f4c3cef)

**Comment by Amrsatrio** [comment]
The last pic shows that it is working?????

**Reply by christopbader** [reply]
Yes, i used Windhawke to get the legacy dialog back and to use to make the screenshot for my comment, I don't understand why this happened as EP is the only customizing tool i use, I installed Windhawke yesterday,  This issue has only been happening for about a week now

**Comment by pyrates999** [comment]
I tested this on a windows 10 install (19045.5011) using EP version 22621.431767.1 with always use legacy transfer dialog box ticked:

![image](https://github.com/user-attachments/assets/82af59c4-d0f4-4ef0-83f0-da150bdee6bd)

and I too didn't see the windows 7 style legacy file transfer dialog box:

![image](https://github.com/user-attachments/assets/a4710c0e-e9e7-4883-9eda-cb685cfb7de0)

I even tried killing all explorer.exe processes and running explorer.exe again.  Same result.

this windhawk mod is what worked though:

![image](https://github.com/user-attachments/assets/7b1d7d4f-6ded-47c4-8d0c-3d3b688eec80)

**Reply by christopbader** [reply]
Thanks for your reply I thought it was a good idea to let people know about this issue
